joplin/api/schema.py

```diff
@@ -189,6 +191,15 @@ class Meta:
     def resolve_janis_urls(self, info):
         return self.specific.janis_urls()
 
+    def resolve_page_type(self, info):
+        return self.content_type
```

Comment:
Let me know if there's a better home or way to manage this. I poked around, but this was the only built-out way i could include each individual pageType for a single allPages query.

Reply:
this is great b/c I think janis needs this too